gulpfile.js

```diff
@@ -0,0 +1,41 @@
+const gulp = require("gulp");
```

Comment:
a little comment explaining what this gulp file purpose is? assume that somebody opens this file and ask you: "What is this file for?"